docs/sphinx/_ext/sphinx-mdx-builder/sphinxcontrib/mdxbuilder/writers/mdx.py

```diff
@@ -331,8 +333,19 @@ def visit_Text(self, node: nodes.Text) -> None:
         if "This parameter will be removed" in content:
             return
 
+        # Prevents wrapped lines in list items, for example in parameter descriptions:
+        #
+        # Args:
+        #     group_name (Optional[str], optional): The name of the asset group.
+        #     dagster_airbyte_translator (Optional[DagsterAirbyteTranslator], optional): The translator to use
+        #         to convert Airbyte content into :py:class:`dagster.AssetSpec`.
+        #         Defaults to :py:class:`DagsterAirbyteTranslator`.
+        if self.in_list_item:
+            content = content.replace("\n", " ")
```

Comment:
Added attribute to track if we are in a list item, and remove new line characters if we are.